.github/workflows/apidoc.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        apiGroup: [ 'management-api', 'control-api' ]
+        apiGroup: [ 'public', 'management-api', 'control-api' ]
```

Comment:
maybe `public-api` would be more consistent with the existing ones?
I agree, that suffix is not really useful, but maybe it could eventually be slashed in a subsequent issue/PR